recipe_recommender/bodybuilding.py

```diff
@@ -94,3 +101,95 @@ def save_df():
 
     # Save as csv
     df.to_csv(save_path)
+
+
+def process_ingredients(data):
+    """
+       Process the ingredients from bodybuilding.com and save the recipe
+
+       data is the scraped json.
+
```

Comment:
Add description of output here. This function creates two files - `ingredients.csv` and `recipe_ingredients.csv` - what's the purpose/difference of these files?